packages/flutter_tools/test/src/common.dart

```diff
@@ -116,6 +116,8 @@ Future<String> createProject(Directory temp, { List<String> arguments }) async {
   final CreateCommand command = CreateCommand();
   final CommandRunner<void> runner = createTestCommandRunner(command);
   await runner.run(<String>['create', ...arguments, projectPath]);
+  // Created `.packages` since it's not created when the flag `--no-pub` is passed.
+  fs.file(fs.path.join(projectPath, '.packages')).createSync();
```

Comment:
Would this not fail for users?

Reply:
In the test, we pass the `--no-pub` flag to `CreateCommand`, so we don't trigger network requests. In a separate PR, it makes sense to mock the service that deals with pub, so it doesn't trigger network requests. This way, we test more code.